### PR TITLE
Rn inventory cart and login flash fix

### PR DESCRIPTION
**Pull request by raymondknguyen**
## Description
Completes User Story #

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which maintains existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## RSpec Results
``

Finished in 13.96 seconds (files took 5.03 seconds to load)
195 examples, 0 failures, 6 pending

Coverage report generated for RSpec to /Users/thunderwolf/turing/mod2_2/projects/monster_shop_1911/coverage. 2992 / 3114 LOC (96.08%) covered.```

### CODE REVIEW

File: app/views/items/edit.html.erb [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <%= text_field_tag :image, "#{@item.image}" %>
 
     <%= label_tag :inventory %>
-    <%= text_field_tag :inventory, "#{@item.inventory}" %>
+    <%= number_field_tag :inventory, "#{@item.inventory}" %>
```
Comment (d-atkins) [inline app/views/items/edit.html.erb:18]: I think this kind of change should be reflected in the `merchant/items` view, which is where items should be editable from

File: spec/features/items/edit_spec.rb [hunk 2]
```diff
@@ -77,6 +77,29 @@
         expect(page).to have_content("Name can't be blank and Image can't be blank")
         expect(page).to have_button("Update Item")
       end
+
+      it "can only enter a number in inventory" do
```
Comment (d-atkins) [inline spec/features/items/edit_spec.rb]: this should be tested in `merchant/items`

File: app/views/merchant/items/edit.html.erb [hunk 3]
```diff
@@ -15,7 +14,7 @@
     <%= text_field_tag :image, "#{@item.image}" %>
 
     <%= label_tag :inventory %>
-    <%= text_field_tag :inventory, "#{@item.inventory}" %>
+    <%= number_field_tag :inventory, "#{@item.inventory}" %>
```
Comment (iEv0lv3) [inline app/views/merchant/items/edit.html.erb:17]: This is the keeper.

**Review by iEv0lv3 — APPROVED** [review]
Okay, I think this PR is good to merge. Tests are good, updates are good.

**Pull request merged**